src/platform/testing/unit/react-testing-library-helpers.js

```diff
@@ -18,15 +18,21 @@ import { commonReducer } from 'platform/startup/store';
  */
 export function renderInReduxProvider(
   ui,
-  { initialState = {}, reducers = {}, ...renderOptions } = {},
+  { initialState = {}, reducers = {}, store = null, ...renderOptions } = {},
```

Comment:
I would like to be able to pass in a store, so that I can use it with set up code before rendering the code being tested.

Reply:
FYI @erikphansen